### ISSUE TITLE
Alert popup - leaving a community on ios

### ISSUE THREAD

**Issue by kingalg** [TryQuiet/quiet#2380]
Version: 2.1.2-alpha.13 (366)
System: iOS

Issue: When I left the community, I got this popup with an alert. I repeated it a few times and wasn't able to recreate it, but as per Wiktor's quick check, it seems important, as its certificate store has a problem. 

![1160D3E3-2592-478E-A500-B085DF65C6E0](https://github.com/TryQuiet/quiet/assets/68909168/a761df01-13c9-47a1-9d2e-4029d388737b)

**Comment by holmesworcester** [comment]
@leblowl  what do you think?

@kingalg does this happen every time you leave? Does it happen even when nobody has joined? If so let's make a test for it!

**Comment by kingalg** [comment]
@holmesworcester no, as I've written in an initial that wasn't able to reproduce it, and since then, I've tried 6 times (leaving as an owner and as a user) with no luck.

**Comment by leblowl** [comment]
Seems like we might not be closing things down correctly when leaving the community. I'm not sure if this is a new issue, we might just have better visibility when it would normally crash, but I can look into it a bit and see. It would be nice to see the whole stack trace, but I think that's a limitation of using `alert` like this.

**Comment by holmesworcester** [comment]
Is adding alert like this a new change for getting better logs on mobile or were we already doing this?

If it's not easily reproducible I don't think it should block release. Moving to mobile backlog.

**Comment by holmesworcester** [comment]
Also I think it makes sense to write a mobile e2e test that catches this and see how often the test can catch this bug.

**Comment by leblowl** [comment]
Adding an alert is a temporary approach for displaying crash information on iOS. It was merged into the 2.1.2 branch to help us debug the instability lately, but planning to remove it before releasing 2.1.2 in favor of a more permanent debug popup (that includes full stack trace and ability to copy/paste).

**Comment by kingalg** [comment]
@leblowl @holmesworcester 
We recently fixed leaving a community (it's working correctly in 2.2.0 prod). We no longer have a problem leaving a community, so the initial issue is resolved. Popup alert was a temporary solution to help identify the cause. Is it still there, Lucas? Either way, as the issue causing this particular popup to appear is resolved, I suggest the following:

1. Closing this issue.
2. Opening an issue for creating a more permanent debug popup (if it's not done yet).

**Comment by kingalg** [comment]
@holmesworcester I think we can close this one; let me know if you agree.
